src/pages/Game.js

```diff
@@ -173,22 +170,53 @@ export default function Game() {
         .map(() => Array(7).fill(null))
     );
     setWinningCells([]);
-
-    // Alterner le joueur qui commence après chaque partie
     const nextStartingPlayer = nextPlayer === "red" ? "yellow" : "red";
     setCurrentPlayer(nextStartingPlayer);
-    setNextPlayer(nextStartingPlayer); // Mettre à jour pour la prochaine partie
-    resetTimer(); // Reset timer on grid reset
+    setNextPlayer(nextStartingPlayer); 
+    resetTimer(); 
+  };
+
+  const handleMenu = () => {
+    setIsOpen(true);
+    setIsPaused(true);
+  };
+
+  const handleClose = () => {
+    setIsOpen(false);
+    setIsPaused(false);
+  };
+
+  const handleRefresh = () => {
+    window.location.reload();
```

Comment:
```suggestion
setNextPlayer("red")
setWinningCells([])
setPoints({ red: 0, yellow: 0 });
 setTimer(15);
 setIsOpen(false);
 setIsPaused(false);
```

c'est pas mieux que le refresh de page pour éviter que la musique coupe ?